src/util/components/models-downloaded/models-downloaded-helper.ts

```diff
@@ -11,6 +11,7 @@ export function postProcessingModelsDownload(modelsDownloaded: ModelsDownloaded[
 export function postProcessingZeroShotEncoders(zeroShotModels: ModelsDownloaded[], encoders: ModelsDownloaded[]): any {
     const prepareModelsList = encoders.filter((el: any) =>
         el.configString != 'bag-of-characters' && el.configString != 'bag-of-words' && el.configString != 'tf-idf');
-    zeroShotModels.sort((a, b) => a.prio - b.prio);
-    return prepareModelsList.concat(zeroShotModels);
+    const zeroShotModelsCopy = [...zeroShotModels];
```

Comment:
not sure why this was changed, but i posted a screenshot that the ordering of the zero-shot models doesn't work as expected, maybe related to this?